salt/iiif/loris.sls

```diff
@@ -75,12 +75,12 @@ loris-dependencies:
             - loris-repository
             - pkg: loris-dependencies
 
-# lsh 2019-03-19: why a special 'loris' user and not www-data?
```

Comment:
I think it was ported over from the documentation of the project

Reply:
yeah, it was just me leaving open questions to my future-self really. Saves me asking them again later ;)